scripts/linters/python_linter.py

```diff
@@ -117,6 +117,44 @@ def _check_import_order(self):
                 summary_messages.append(summary_message)
         return summary_messages
 
+    def _check_non_test_files(self):
+        """This function is used to check that function
+           with test_only in their names are in test files.
+        """
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting function defintion checks')
+            python_utils.PRINT('----------------------------------------')
+        summary_messages = []
+        files_to_check = self.py_filepaths
+        with linter_utils.redirect_stdout(sys.stdout):
+            failed = False
+            for filepath in files_to_check:
+                if not filepath.endswith('_test.py'):
+                    file_content = FILE_CACHE.read(filepath)
```

Comment:
Use `readlines` instead

Reply:
umm, but in last comment you said to refer js_ts_linter files, there every function is using .read()
and also , this change for code quality or there is something buggy?

Reply:
Code quality

Reply:
done